app/views/layouts/_vertical_separator.html.erb

```diff
@@ -0,0 +1,3 @@
+<svg id="vertical-separator" width="3" height="22">
+  <line x1="1.7" y1="0" x2="1.7" y2="22"></line>
+</svg>
```

Comment:
We don#t normally use svg, because legacy browsers. You could use a left border on an inline-div?

Reply:
We did it before, at some point, in one of the alphas. Blanca will remember.